### DISCUSSION TITLE
displaying .gif in image-keyboard-response

### DISCUSSION THREAD

**Discussion by nrschoen** [jspsych/jsPsych#1591] [Q&A] [answered]
Hi,

I am making an experiment with small animations in gif format and it worked fine a couple days ago. However, now it shows the gif as a static gif. I already tried preloading and also not using a timeline variable but directly putting the image path into the stimulus field. Here is my code:

![Screen Shot 2021-03-01 at 17 24 52](https://user-images.githubusercontent.com/65179042/109534601-6ceae480-7ab3-11eb-9ad8-8dc2542bd1fc.png)

Thanks in advance!

Best,
Neil

**Comment by jodeleeuw** [comment, marked as answer]
Hi Neil,

This is probably because of a change to our `image-keyboard-response` plugin in jsPsych 6.2/6.3. Try setting `render_on_canvas: false` in the parameters of that trial.

**Reply by nrschoen** [reply]
Thank you very much. Perfect as always!

**Reply by jodeleeuw** [reply]
Great! @becky-gilbert we should probably note that animated .gifs aren't compatible with canvas rendering in the docs of the image-* plugins at some point.

**Reply by becky-gilbert** [reply]
Ugh, sorry about this @nrschoen. I totally forgot about animated gifs when we added the canvas rendering option ☹️Glad this was an easy fix! 
@jodeleeuw I'll add this to the image-* plugin docs for v6.3.1.
